src/components/Toasts/ToastController.tsx

```diff
@@ -0,0 +1,88 @@
+import toast from "react-hot-toast";
+import TransactionToast from "./TransactionToast";
+import GenericToast from "./GenericToast";
+
+const formatDate = (date: Date) => {
```

Comment:
`const formatDate = (date: Date): string => {`

Move this function to Toasts/helpers/formatDate.ts and write some tests in Toasts/helpers/formatDate.test.ts